fluid/sfont.cpp

```diff
@@ -80,12 +80,23 @@ bool SFont::read(const QString& s)
       if (!load())
             return false;
 
-      foreach(Instrument* i, instruments) {
+      synth->setLoadProgress(0);
+      int currentInstument = 0;
+      foreach(Instrument* i, instruments)
```

Comment:
Opportunity to change the foreach to a for ?
The indent of { } is not good. See https://musescore.org/en/handbook/developers-handbook/finding-your-way-around/musescore-coding-rules